### PR TITLE
Docs/install changes

### PR DESCRIPTION
**Pull request by bgunnar5**
I went through all of the docs and added some new instructions where I felt things weren't as clear as they could be. I also added instructions on how to install merlin via make for developers and fixed typos when I found them.

This branch was created off of the fix_footer branch in order to work with the newer version of Sphinx so the commits from Nov 2, 2022 (commit id d10510d) are the same as in the fix_footer branch that's waiting to be merged.

### CODE REVIEW

File: docs/source/merlin_variables.rst [hunk 1]
```diff
@@ -31,73 +31,138 @@ The directory structure of merlin output looks like this:
...
+
+            /*/*/*/*
+ 
+   * - ``$(MERLIN_PATHS_ALL)``
+     - A space delimited string of all of the paths;
+       can be used as is in bash for loop for instance with:
+ 
+       .. code-block:: bash
+ 
+          for path in $(MERLIN_PATHS_ALL)
```
Comment (lucpeterson) [inline docs/source/merlin_variables.rst:115]: One of these blocks shouldn't be here

File: docs/source/modules/hello_world/hello_world.rst [hunk 2]
```diff
@@ -234,6 +240,12 @@ Immediately after that, this will pop up:
 .. literalinclude :: celery.txt
    :language: text
 
+You may not see all of the info logs listed after the Celery C is displayed. If you'd like to see them you can change Merlin's log levels with the ``-lvl`` or ``--level`` tag. For more information on this, you can run:
```
Comment (lucpeterson) [inline docs/source/modules/hello_world/hello_world.rst:243]: this is actually changed via the workers' log level, not merlin's

**Review by lucpeterson — CHANGES_REQUESTED** [review]
just a few comments, but this is a huge improvement!

File: docs/source/modules/hello_world/hello_world.rst (hunk 2 above)
Comment (bgunnar5) [reply docs/source/modules/hello_world/hello_world.rst:243]: Do you have an example of how to change their log levels via the CLI? When I run `merlin run-workers --help` the optional arguments don't show a log level flag to set. The only optional flags that are listed for run-workers are: --help, --worker-args, --steps, --echo, and --vars.

Comment (bgunnar5) [reply docs/source/modules/hello_world/hello_world.rst:243]: @lucpeterson (not sure if it notifies you when I comment so figured I'd add this @ here)

**Review by lucpeterson — APPROVED** [review]
Does this PR make #382 now obsolete?

**Pull request merged**